ntp-proto/src/keyset.rs

```diff
@@ -170,24 +164,27 @@ impl KeySet {
     }
 
     pub(crate) fn encode_cookie(&self, cookie: &DecodedServerCookie) -> Vec<u8> {
-        let mut plaintext = Self::plaintext(cookie);
-        let plaintext_len = plaintext.as_slice().len();
-
-        let (siv_tag, nonce) = self.keys[self.primary as usize]
-            .encrypt_in_place_detached(plaintext.as_mut(), &[])
+        let mut output = Self::plaintext(cookie);
```

Comment:
I think we now can/should make this a method on `DecodedServerCookie`.

Reply:
agreed, done